src/test/java/core/basesyntax/service/impl/CsvParserServiceTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvParserServiceTest {
+    private CsvParserService csvParserService;
+
+    @BeforeEach
+    void setUp() {
+        csvParserService = new CsvParserService();
+    }
```

Comment:
for all test classes:
try to initialize services required for testing once (during declaration or in BeforeAll method) as when test-suite becomes bigger unnecessary creation of objects affects time to run tests